### PR TITLE
16 add button load pokemon list

### PR DESCRIPTION
**Pull request by bdtkevin**

### CODE REVIEW

**Review by RobinDela — DISMISSED** [review]
Clean

File: src/components/PokemonList.jsx [hunk 1]
```diff
@@ -32,6 +41,17 @@ const PokemonList = ({
         onChange={handleQueryChange}
         placeholder=" Search your Pokemon by name..."
       />
+      <select name="list-choice" id="list-choice" onChange={handleList}>
+        <option>1-99</option>
```
Comment (LoicBrassart) [inline src/components/PokemonList.jsx]: `<option value="0">` would be **much** better !

File: src/components/PokemonList.jsx [hunk 2]
```diff
@@ -12,17 +12,26 @@ const PokemonList = ({
...
+
+  const handleQueryChange = (e) => {
+    setQuery(e.target.value);
+  };
+
+  const handleList = (e) => {
+    const choice = e.target.value;
+
+    setQuery('');
+    setOffset(parseInt(choice.slice(0, choice.indexOf('-')), 10) - 1);
```
Comment (LoicBrassart) [inline src/components/PokemonList.jsx]: This bit of code is overkill, just use the `value` attribute of your `option` !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
A bit of algo to update

File: src/components/PokemonList.jsx (hunk 1 above)
Comment (bdtkevin) [reply src/components/PokemonList.jsx]: 👍

File: src/components/PokemonList.jsx (hunk 2 above)
Comment (bdtkevin) [reply src/components/PokemonList.jsx]: 👍

**Review by RobinDela — APPROVED** [review]
Good job

**Review by LoicBrassart — APPROVED** [review]
No more Evil !

**Pull request merged**